src/PROPOSAL/PROPOSAL/secondaries/parametrization/photopairproduction/PhotoPairProductionTsai.h

```diff
@@ -30,5 +30,43 @@ namespace secondaries {
                 const Vector3D&, double, double, const Component&,
                 double, double, double) override;
     };
+
+    class PhotoPairProductionTsaiSauter : public PhotoPairProductionTsaiForwardPeaked {
```

Comment:
Is this the exact same as `PhotoPairProductionKochMotzSauter` and if yes, can it be implemented to avoid duplication?